.github/workflows/CI.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.8'
+          - '1.9'
```

Comment:
We should add 1.10 for testing; current `nightly` is 1.11.0-DEV, I think.

Reply:
Julia v1.9 is needed to use weak dependencies. We will have to wait a bit longer for 1.10 to be out; at least having nightly working is a good indicator (due to backward compatibility) that it should also pass for 1.10.